mteb/tasks/Classification/slk/SlovakHateSpeechClassification.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class SlovakHateSpeechClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SlovakHateSpeechClassification",
+        description="The dataset contains posts from a social network with human annotations for hateful or offensive language in Slovak.",
+        reference="https://huggingface.co/datasets/TUKE-KEMT/hate_speech_slovak",
+        dataset={
+            "path": "TUKE-KEMT/hate_speech_slovak",
+            "revision": "f9301b9937128c9c0b636fa6da203aeb046479f4",
+        },
+        type="Classification",
+        category="s2s",
+        modalities=["text"],
+        date=("2024-05-25", "2024-06-06"),
```

Comment:
Are all the posts collected from the same date??

Reply:
The problem is that I don't have the information about the date. So I just put the date when we created the dataset.